full/src/test/java/apoc/export/csv/ExportXlsTest.java

```diff
@@ -71,6 +77,39 @@ public void testExportGraphXls() throws Exception {
         assertExcelFileForGraph(fileName);
     }
 
+    @Test
+    public void testExportGraphXlsWithMoreThan100Nodes()  {
+        db.executeTransactionally("UNWIND range(1,200) as range CREATE (n:Test)");
+        String fileName = "graph.xlsx";
+        TestUtil.testCall(db, "CALL apoc.graph.fromDB('test',{}) yield graph " +
+                        "CALL apoc.export.xls.graph(graph, $file,null) " +
+                        "YIELD nodes, relationships, properties, file, source,format, time " +
+                        "RETURN *",
+                map("file", fileName),
+                (r) -> assertResults(fileName, r, "graph", 208L, 2L, 206));
+        assertExcelFileForGraph(fileName);
+        db.executeTransactionally("MATCH (n:Test) DETACH DELETE n");
+    }
+
+    @Test
+    public void testExportGraphXlsWithCustomHeaderAndMoreThan100Nodes() {
+        String nodeId = "customNode";
+        String relId = "customRel";
+        String startNodeId = "customStart";
+        String endNodeId = "customEnd";
+        db.executeTransactionally("UNWIND range(1,200) as range CREATE (n:Test)");
+        String fileName = "graph.xlsx";
+        TestUtil.testCall(db, "CALL apoc.graph.fromDB('test',{}) yield graph " +
+                        "CALL apoc.export.xls.graph(graph, $file, $conf) " +
+                        "YIELD nodes, relationships, properties, file, source,format, time " +
+                        "RETURN *",
+                map("file", fileName, "conf", map("headerNodeId", nodeId,
+                        "headerRelationshipId", relId, "headerStartNodeId", startNodeId, "headerEndNodeId", endNodeId)),
+                (r) -> assertResults(fileName, r, "graph", 208L, 2L, 206));
```

Comment:
can you explain these numbers?
Shouldn't we get 200 nodes, 0 rels?
Not sure what 206 counts, here.

Reply:
The number `206` is due to `setUp()` method, where 6 additional nodes are created (the same for relationships).
The `208L` instead it is due to having 2 nodes with double labels (also in `setUp()`).
So, this numbers are the sum of `assertResults` on line `177` plus 200 new nodes.